src/components/VBtn/VBtn.vue

```diff
@@ -1,14 +1,18 @@
 <script>
 /**
- * @param {{ to: string }} props
- * @param {{ attrs: { href: string } }} data
- * @return { string }
+ * Detects if a VDOM element is a <RouterLink>, <a>, or <button>
+ *
+ * @param  {object} props           props container
+ * @param  {string} props.to        the :to prop for router-link
+ * @param  {object} data            data container
+ * @param  {object} data.attrs      attributes container
```

Comment:
Im fine with using the more verbose syntax for this, but in this case we lost the typing for `href`

```suggestion
 * @param  {object} data.attrs      attributes container
 * @param  {string} data.attrs.href      href attribute
```